redisson/src/main/java/org/redisson/api/RedissonClient.java

```diff
@@ -475,6 +475,30 @@
      */
     RLock getLock(String name);
 
+    /**
+     * Returns Spin lock instance by name.
+     * <p>
+     * Implements a <b>non-fair</b> locking so doesn't guarantees an acquire order by threads.
+     * <p>
+     * Lock doesn't use a pub/sub mechanism
+     *
+     * @param name - name of object
+     * @return Lock object
+     */
+    RLock getSpinLock(String name);
+
+    /**
+     * Returns Spin lock instance by name with specified back off options.
+     * <p>
+     * Implements a <b>non-fair</b> locking so doesn't guarantees an acquire order by threads.
+     * <p>
+     * Lock doesn't use a pub/sub mechanism
+     *
+     * @param name - name of object
+     * @return Lock object
+     */
+    RLock getSpinLock(String name, LockOptions.BackOffOptions backOffOptions);
```

Comment:
We need retain just backoff word without options. Since it's already belong LockOptions object